truffle.js

```diff
@@ -0,0 +1,44 @@
+require("babel-polyfill");
+require("babel-register")({
+  "presets": ["es2015"],
+  "plugins": ["syntax-async-functions", "transform-regenerator"]
+});
+
+module.exports = {
+  networks: {
+    live: {
+      host: "localhost",
+      port: 8545,
+      network_id: "1",
+      gas: 4543760
```

Comment:
this number makes no sense, maybe its not used but then shouldn't we just remove?

Reply:
it is the gas limit for deployment .
It is optional with a default value of 4712388.
we might get along without that.
http://truffleframework.com/docs/advanced/configuration#networks

Reply:
isn't it way low?

Reply:
it is low for genesis scheme which need around 6500000 .so we set the gas for this contract deployment  separately ..
For all other contracts it is ok .so far.

Reply:
it has worked so far for me doing deployments.